utils/validators.js

```diff
@@ -30,7 +30,7 @@ module.exports.register = ({ name, email, password, password2 }) => {
 	return { errors, isValid: isEmpty(errors) };
 };
 
-module.exports.createHabit = ({ name, description, type, difficulty, tags, frequency, user }) => {
+module.exports.createHabit = ({ name, description, type, difficulty, tags, frequency }) => {
```

Comment:
Why remove `user`? 😕

Reply:
My train of thought was that the only way `req.user` can be there is if it has been authenticated with passport, which sort of validates it in itself... right?

Its no trouble passing that through the validator as well, though. I'll do it now 😄